File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    
    l_1 = len(arr1)
    l_2 = len(arr2[0])
    answer= [[0 for _ in range(l_2)] for _ in range(l_1)]
    
    for i in range(l_1):
        for j in range(l_2):
            for idx in range(len(arr2)):
                answer[i][j] += (arr1[i][idx] * arr2[idx][j]) 
                
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
  const result = [];
  for (let i = 0; i < arr1.length; i++) {
    const row = [];
    for (let j = 0; j < arr2[0].length; j++) {
      let sum = 0;
      for (let k = 0; k < arr2.length; k++) {
        sum += arr1[i][k] * arr2[k][j];
      }
      row.push(sum);
    }
    result.push(row);
  }
  return result;
}